Cargo.toml

```diff
@@ -45,7 +45,7 @@ structopt = "~0.3.9"
 
 [features]
 mock_base = [
-    "lru_time_cache/fake_clock",
+    "lru_time_cache/sn_fake_clock",
```

Comment:
the feature name also changed to snake ?

Reply:
@maqi yes, for clarity & consistency I updated that in lru_time_cache as well - see [here](https://github.com/maidsafe/lru_time_cache/commit/6b3cf411ea2b26596bdf87659e3afe967b2ebdff).
(And I published a new lru version)